springfox-core/src/main/java/springfox/documentation/service/ResolvedMethodParameter.java

```diff
@@ -31,20 +33,31 @@
 
 public class ResolvedMethodParameter {
   private final int parameterIndex;
+  private final boolean returnType;
   private final List<Annotation> annotations;
   private final String defaultName;
   private final ResolvedType parameterType;
 
   public ResolvedMethodParameter(String paramName, MethodParameter methodParameter, ResolvedType parameterType) {
     this(methodParameter.getParameterIndex(),
+        false,
         paramName,
         newArrayList(methodParameter.getParameterAnnotations()),
         parameterType);
   }
+  
+  public ResolvedMethodParameter(ResolvedType parameterType) {
+    this(0,
+        true,
+        "",
+        newArrayList(new Annotation[0]),
+        parameterType);
+  }
 
-  public ResolvedMethodParameter(int parameterIndex, String defaultName, List<Annotation> annotations, ResolvedType
+  private ResolvedMethodParameter(int parameterIndex, boolean returnType, String defaultName, List<Annotation> annotations, ResolvedType
```

Comment:
Ok, but I needed some wrapper for the return parameter. For now, the only information that is related to return parameter is ResolvedType. But when we add @JsonView we will need some stuff to ship view groups, and I was planning to add this groups to ResolvedMethodParameter and then to ModelContext. (That is one of my ideas...).  Nevertheless, I think, that is more convinient to use one type to Introduce all parameters (inbound and outbound) with related additional information.